charts/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: sendgrid-stats-exporter
-description: A Helm chart for chatwork/sendgrid-stats-exporter
+description: A Helm chart for ubie-oss/sendgrid-stats-exporter
 type: application
 version: 0.0.8
 appVersion: 0.0.8
```

Comment:
I'm not sure if it will be a problem, but using the same version as theirs may conflict in the future. Should we re-version ours, or do you have some idea to avoid version conflicts by applying their changes?

Reply:
OK. I don't know if there will be a problem, and I'm not going to investigate it in detail, so I'll set the major version to 2 for now. If we do it strictly, it is better to separate the namespace, but the current name clearly expresses the process and I do not want to change it.